### PR TITLE
Move SSL configuration to separate repo

### PR DESCRIPTION
**Pull request by Reception123**
Moved to Miraheze/ssl

### CODE REVIEW

File: modules/mediawiki/manifests/wikistats.pp [hunk 1]
```diff
@@ -8,7 +8,7 @@
         group  => 'www-data',
     }
 
-    $customdomains = hiera_hash('ssl')
+    $customdomains = loadyaml('/etc/puppet/ssl/certs.yaml', '/etc/puppet/ssl/redirects.yaml')
```
Comment (JohnFLewis) [inline modules/mediawiki/manifests/wikistats.pp]: custom domains are only real wikis, not redirects.

File: modules/puppetmaster/manifests/init.pp [hunk 2]
```diff
@@ -76,6 +76,11 @@
         directory => '/etc/puppet/parsoid',
         origin    => 'https://github.com/miraheze/parsoid.git',
     }
+	git::clone { 'ssl':
+        ensure    => latest,
+        directory => '/etc/puppet/ssl',
+        origin    => 'https://github.com/miraheze/ssl.git',
+    }
```
Comment (JohnFLewis) [inline modules/puppetmaster/manifests/init.pp]: needs a cron like below (parsoid-git)

File: modules/ssl/manifests/cert.pp [hunk 3]
```diff
@@ -5,7 +5,7 @@
     if defined(File["/etc/ssl/certs/${certificate}.crt"]) {
         file { "/etc/ssl/certs/${certificate}.crt":
             ensure => $ensure,
-            source => "puppet:///modules/ssl/certificates/${certificate}.crt",
+            source => "puppet:///ssl/certificates${certificate}.crt",
```
Comment (JohnFLewis) [inline modules/ssl/manifests/cert.pp]: syntax

File: modules/ssl/manifests/hiera/certs.pp [hunk 4]
```diff
@@ -22,7 +22,7 @@
         file { "${sslurl}":
             ensure => present,
             path   => "/etc/ssl/certs/${sslurl}.crt",
-            source => "puppet:///modules/ssl/certificates/${sslurl}.crt",
+            source => "puppet:///ssl/certificates${sslurl}.crt",
```
Comment (JohnFLewis) [inline modules/ssl/manifests/hiera/certs.pp]: syntax

**Comment by Reception123** [comment]
follow up is #452

**Pull request merged**